packages/reason-relay/language-plugin/src/generator/Printer.re

```diff
@@ -174,19 +174,16 @@ and printObject =
          }
        )
     |> Array.iteri((index, p) => {
```

Comment:
not that you really have perf issues here, but Belt versions of array functions are usually more optimized.

Reply:
Ah nice, I'll change that! It's not totally unlikely that I'll end up porting this to native Reason at some point when the new Relay compiler ships. But let's cross that bridge when we get to it.